docs/contributor/code-contribute.md

```diff
@@ -369,4 +369,14 @@ Learn how to write the docs by the following guide:
 * [kubevela.io Developer Guide](https://github.com/kubevela/kubevela.io/blob/main/README.md)
 * [Update KubeVela.io CLI Reference Doc](./cli-ref-doc)
 
-Great, you have complete the lifecycle of code contribution, try to [join the community as a member](https://github.com/kubevela/community/blob/main/community-membership.md) if you're interested.
\ No newline at end of file
+Great, you have complete the lifecycle of code contribution, try to [join the community as a member](https://github.com/kubevela/community/blob/main/community-membership.md) if you're interested.
+
+## FAQ
+
+### Q: How can I run vela-core locally for debugging?
+ - Firstly, scaling the replicas of `kubevela-vela-core` to 0 for leader election of `controller-manager`, e.g `kubectl scale deploy -n vela-system kubevela-vela-core --replicas=0`.
+ - Secondly, removing the `WebhookConfiguration`, otherwise an error will be reported when applying your application using `vela-cli` or `kubectl`.
+   ```shell
+   Internal error occurred: failed calling webhook 'validating.core.oam.dev.v1beta1.applications': Post "https://vela-core-webhook.vela-system.svc:443/validating-core-oam-dev-v1beta1-applications?timeout=10s"
+   ```
+ - Finally, pulling the repository of KubeVela and debugging the code with breakpoints. 
```

Comment:
This part should be in the code contribution gude

Reply:
> This part should be in the code contribution gude

ok

Reply:
> What's the difference between the current make vela-core run in the section https://kubevela.io/docs/contributor/code-contribute#build

build is about how to run locally, it is mainly about how to debug locally